src/fontcache/freetypefontcache.cpp

```diff
@@ -296,19 +206,116 @@ GlyphID FreeTypeFontCache::MapCharToGlyph(char32_t key, bool allow_fallback)
 	return glyph;
 }
 
-/**
- * Free everything allocated w.r.t. freetype.
- */
-void UninitFreeType()
-{
-	FT_Done_FreeType(_ft_library);
-	_ft_library = nullptr;
-}
+FT_Library _ft_library = nullptr;
+
+class FreeTypeFontCacheFactory : public FontCacheFactory {
+public:
+	FreeTypeFontCacheFactory() : FontCacheFactory("freetype", "FreeType font provider") {}
+
+	virtual ~FreeTypeFontCacheFactory()
+	{
+		FT_Done_FreeType(_ft_library);
+		_ft_library = nullptr;
+	}
+
+	/**
+	 * Loads the freetype font.
+	 * First type to load the fontname as if it were a path. If that fails,
```

Comment:
```suggestion
	 * First try to load the fontname as if it were a path. If that fails,
```
This was already wrong before this PR.